content/blog/2023-09-27-epidatr.Rmd

```diff
@@ -0,0 +1,94 @@
+---
+title: New package released! `epidatr`, an R client for Delphi Epidata API
+author: Dmitry Shemetov, David Weber
+date: 2023-09-27
+tags:
+  - r
+  - epidata
+  - covidcast
+authors:
+  - dmitry
+  - david
```

Comment:
This plain "david" is pointing to David Farrow in the preview (at the bottom of the page).  We need to add a key for David Weber in https://github.com/cmu-delphi/www-main/blob/dev/content/people/index.md, and then use it here.  We could also add biographical blurbs for them in that same file, if desired.
```suggestion
  - davidweb
```